### PR TITLE
fix(autoware_behavior_velocity_crosswalk_module): fix passedByValue

### PR DESCRIPTION
**Pull request by kobayu858**
## Description
This is a fix based on cppcheck constParameterReference warnings

```
planning/behavior_velocity_planner/autoware_behavior_velocity_crosswalk_module/src/occluded_crosswalk.cpp:89:29: performance: Function parameter 'velocity_thresholds' should be passed by const reference. [passedByValue]
  const std::vector<double> velocity_thresholds, const double inflate_size)
                            ^

planning/behavior_velocity_planner/autoware_behavior_velocity_crosswalk_module/src/scene_crosswalk.cpp:148:48: performance: Function parameter 'stop_factor_points' should be passed by const reference. [passedByValue]
  const std::vector<geometry_msgs::msg::Point> stop_factor_points = {})
                                               ^
```
## Related links

**Parent Issue:**

- Link

## Notes for reviewers

None.

## Interface changes

None.

## Effects on system behavior

None.

### CODE REVIEW

File: planning/behavior_velocity_planner/autoware_behavior_velocity_crosswalk_module/src/occluded_crosswalk.hpp [hunk 1]
```diff
@@ -80,7 +80,7 @@ double calculate_detection_range(
 /// @return selected and inflated objects
 std::vector<autoware_perception_msgs::msg::PredictedObject> select_and_inflate_objects(
   const std::vector<autoware_perception_msgs::msg::PredictedObject> & objects,
-  const double velocity_threshold, const bool skip_pedestrians, const double inflate_size);
+  const double & velocity_threshold, const bool skip_pedestrians, const double inflate_size);
```
Comment (soblin) [inline planning/behavior_velocity_planner/autoware_behavior_velocity_crosswalk_module/src/occluded_crosswalk.hpp:83]: In general for literal type like int/double/etc,  passing by value is faster than passing by reference. But it is OK

**Review by soblin — APPROVED** [review]

File: planning/behavior_velocity_planner/autoware_behavior_velocity_crosswalk_module/src/occluded_crosswalk.hpp (hunk 1 above)
Comment (kobayu858) [reply planning/behavior_velocity_planner/autoware_behavior_velocity_crosswalk_module/src/occluded_crosswalk.hpp:83]: Thanks for the advice.

Comment (takayuki5168) [inline planning/behavior_velocity_planner/autoware_behavior_velocity_crosswalk_module/src/occluded_crosswalk.hpp:83]:
```suggestion
  const double velocity_threshold, const bool skip_pedestrians, const double inflate_size);
```

Comment (takayuki5168) [reply planning/behavior_velocity_planner/autoware_behavior_velocity_crosswalk_module/src/occluded_crosswalk.hpp:83]: I reverted this change.

Comment (takayuki5168) [inline planning/behavior_velocity_planner/autoware_behavior_velocity_crosswalk_module/src/occluded_crosswalk.hpp:83]:
```suggestion
  const double velocity_threshold, const bool skip_pedestrians, const double inflate_size);
```

Comment (takayuki5168) [reply planning/behavior_velocity_planner/autoware_behavior_velocity_crosswalk_module/src/occluded_crosswalk.hpp:83]: I agree with @soblin 's comment. I created a suggestion.

Comment (kobayu858) [reply planning/behavior_velocity_planner/autoware_behavior_velocity_crosswalk_module/src/occluded_crosswalk.hpp]: Thank you very much. I have corrected it.

**Review by veqcc — APPROVED** [review]

**Pull request merged**